### DISCUSSION TITLE
elevators/tp

### DISCUSSION THREAD

**Discussion by oskolot** [TechReborn/TechReborn#2855] [Ideas]
i think that having a teleportation block kind of like a instant teleportation between diffrent farms or bases whould be very usefull.
for exampel you could teleport between layers of ur base kind of like an elevator or you can teleport between farms or bases.
i also suggest making it hard to acheive becouse teleportation is pretty op

**Comment by Ayutac** [comment]
concept design already exists accessable to devs only. What we need is mainly a dev with time to implement it, but it is planned as an eventual feature.

**Comment by drcrazy** [comment]
Let's put some concepts here:
Use freq transmitter to link TP block with each other. But you will need 2 pairs on TP blocks for two way transportation if freq transmitter is used to link source to destination. Or destination to source. Which will confuse players.
Should player be teleported as soon as in stood on TP block? ANother option is that player need to crouch to be teleported. This way accidental teleports are prevented, but it might be inconvenient to players with increased movement speed. It will take some time to position on TP block before crouch
Should freq transmitter teleport player to linked block? Usual option is that there are two tiers of TP devices, tier 1 have only one destination. Tier 2 allows to select destination
Also I personally like ideas of elevators. They are limited but accessible quite early game. Like this one - https://www.curseforge.com/minecraft/mc-mods/openblocks-elevator

**Reply by oskolot** [reply]
i think having to crouch is the best option Couse it will feel more controlled when put in a higher tier where you can choose destination.
Another concept of teleporting is having a item that can instantly take you home but you need to have it fully charged to have it actually work then you can add tiers to that making it so u can choose where you teleport whether its ur base or a farm but it will cost alot of energy. you also get a range upgrade as you go up in tiers for exampel tier 1 500 blocks, tier 2 1000 blocks and multiple stations to choose from.
This type of teleport should be very hard to get because you can have 2 devices and stations and not needing a tier 2
i suggest (if added) that it has some sort of limit whether its how many blocks u can tp or how many you can have.

**Comment by xanthian** [comment]
I used quartz elevators mod, which i felt did a decent job of not making it too op.
it could only tp up or down on the same block (X,Z) with a limited range.
crouch to go down, jump to go up

a similar, but more expensive variation of this would be a good start
